webpack/assets/javascripts/react_app/components/toastNotifications/toastListitem/Toast.scss

```diff
@@ -0,0 +1,9 @@
+.toast-pf.alert {
+  .close {
```

Comment:
you're right, i've removed the margin, but the outline still bugs me so i'll keep it